apps/notifications/src/app/note-list/index.tsx

```diff
@@ -57,10 +61,6 @@ const NoteList = ( { filterName }: { filterName: keyof ReturnType< typeof getFil
 		}
 	}, [ client, isLoading ] );
 
-	useEffect( () => {
```

Comment:
This change is required. Otherwise, the scroll restoration won't work for items after > 10, because the component will always first render 10 items before this effect is run...